src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1306,15 +1306,15 @@ public virtual ChallengeResult Challenge(AuthenticationProperties properties)
             => new ChallengeResult(authenticationSchemes, properties);
 
         /// <summary>
-        /// Creates a <see cref="ForbidResult"/>.
+        /// Creates a <see cref="ForbidResult"/> (HTTP 403).
```

Comment:
FYI, though a 403 response is what happens by default when calling this method, there's no guarantee about that, as the status code is actually specific to the handler that handles the `ForbidResult`.
For instance, the cookies middleware will redirect the user to the "access denied page" (302) instead of returning an empty 403 response.